src/manual/endpoint-kafka.adoc

```diff
@@ -189,52 +190,63 @@ The following table shows all available settings on a Kafka endpoint in Citrus:
 
 | consumer-group
 | No
-| citrus_kafka_group
-| Consumer group name. Please keep in mind that records are load balanced across consumer instances with the same consumer
-  group name set. So you might run into message timeouts when using multiple Kafka endpoints with the same consumer group name.
+| `citrus_kafka_group`
+| Consumer group name.
+Please keep in mind that records are load balanced across consumer instances with the same consumer group name set.
+So you might run into message timeouts when using multiple Kafka endpoints with the same consumer group name.
+
+| random-consumer-group
```

Comment:
```suggestion
| `random-consumer-group`
```

Reply:
I've corrected this for all properties, not only the random consumer group.